src/compressed_tensors/offload/dist_utils.py

```
@@ -7,18 +7,34 @@
 import torch.distributed as dist
 
 
-__all__ = ["is_distributed", "is_rank0"]
+__all__ = ["is_distributed", "is_rank0", "init_dist"]
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)